### DISCUSSION TITLE
Is it possible to use the app on mobile?

### DISCUSSION THREAD

**Discussion by blekster** [SebOuellette/LiveBot#215] [Q&A]
Hello! I was wondering if it is in any way possible to run LiveBot on a mobile device (Android specifically), as my goal with this app is to use the bots as role play characters in a private server and it'd be very useful to have the possibility on the go. I tried using a couple apps to run the .exe file, but haven't been successful yet. Thank you in advance!

**Comment by thecatontheceiling** [comment]
probably no

**Reply by Goergegamingroblox** [reply]
Rip

**Comment by rabbanyhmm** [comment]
> Hello! I was wondering if it is in any way possible to run LiveBot on a mobile device (Android specifically), as my goal with this app is to use the bots as role play characters in a private server and it'd be very useful to have the possibility on the go. I tried using a couple apps to run the .exe file, but haven't been successful yet. Thank you in advance!

LiveBot doesn’t support Android. You can try using remote desktop apps, but no native mobile version. 😕
